index.js

```diff
@@ -1,88 +1,128 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+counter++
+
+console.log(counter) 
+//=> 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+const name = "Jaime"
+console.log(name)
+//=> "Raffy"
+//name = "Not Raffy" 
+//=> TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+console.log(drinkWater(12))
+function drinkWater(thirstLevel){
+// => "Man I sure am thirsty"
+console.log("Mann I sure am thirsty")
+thirstLevel -= 1
+// => "Ahh that hits the spot"
+console.log("Ahh that hits the spot")
+console.log(thirstLevel)
```

Comment:
Nice! Just remember, this function should be returning the `thirstLevel` and returning a value is different than just console logging a value
```suggestion
return thirstLevel
```

Reply:
- If we don't have the `return` and we try to use this function for getting a value, it will not work as expected as demonstrated in the following example:

```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // undefined
```

- if we do have the `return` though, the same exact code will yield different results as shown below:
```javascript
let myThirstLevel = drinkWater(7)
console.log(myThirstLevel) // 6
```